### PR TITLE
Update sessions page columns, add column selector.

### PR DESCRIPTION
**Pull request by stbof**
https://cockroachlabs.atlassian.net/browse/DOC-2704
https://cockroachlabs.atlassian.net/browse/DOC-2637
https://cockroachlabs.atlassian.net/browse/DOC-3394

### CODE REVIEW

**Review by kevin-v-ngo — APPROVED** [review]
Looks good!

File: _includes/common/ui/sessions-page.md [hunk 1]
```diff
@@ -6,77 +6,6 @@
   {% assign page_prefix = "ui-" %}
 {% endif %}
 
-## Sessions list
+## Sessions table
```
[the remark this answers is not in the document]
Comment (stbof) [reply _includes/common/ui/sessions-page.md:9]: This is referring to the table containing the data, not the tab itself.

File: _includes/v21.2/ui/sessions.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+The following are displayed for each active session:
+
+Column | Description
+--------- | -----------
+Session Duration | Amount of time the session has been open.
+Transaction Duration | Amount of time the transaction has been active, if there is an open transaction. If a session is idle, this column will display `N/A`.
+Statement Duration | Amount of time the SQL statement has been active, if there is an active statement. If a session is idle, this column will display `N/A`.
+Memory Usage | Amount of memory currently allocated to this session, followed by the maximum amount of memory this session has ever been allocated.
+Statement | Active SQL statement. If more than one statement is active, the most recent statement is shown. If a session is idle, this column will display `N/A`.
```
[the remark this answers is not in the document]
Comment (stbof) [reply _includes/v21.2/ui/sessions.md]: Fixed.

**Review by kevin-v-ngo — APPROVED** [review]
Looks good! :)

File: v22.1/ui-sessions-page.md [hunk 3]
```diff
@@ -6,11 +6,13 @@ docs_area: reference.db_console
 ---
 
 {{site.data.alerts.callout_info}}
-On a secure cluster, this area of the DB Console can only be accessed by a SQL user with the `VIEWACTIVITY` role option. Note that non-`admin` users will see only their own sessions, while `admin` users see sessions for all users.
+On a secure cluster, this area of the DB Console can only be accessed by a SQL user with the `VIEWACTIVITY` [role option](alter-role.html#role-options). Non-`admin` users will see only their own sessions, while `admin` users see sessions for all users.
 {{site.data.alerts.end}}
 
-The **Sessions** page provides details of all open sessions in the cluster.
+The **Sessions** page  of the DB Console provides details of all open sessions in the cluster.
```
Comment (maryliag) [inline v22.1/ui-sessions-page.md:12]: same comment as above

File: v21.2/ui-sessions-page.md [hunk 4]
```diff
@@ -6,11 +6,13 @@ docs_area: reference.db_console
 ---
 
 {{site.data.alerts.callout_info}}
-On a secure cluster, this area of the DB Console can only be accessed by a SQL user with the `VIEWACTIVITY` role option. Note that non-`admin` users will see only their own sessions, while `admin` users see sessions for all users.
+On a secure cluster, this area of the DB Console can only be accessed by a SQL user with the `VIEWACTIVITY` [role option](alter-role.html#role-options). Non-`admin` users will see only their own sessions, while `admin` users see sessions for all users.
 {{site.data.alerts.end}}
 
-The **Sessions** page provides details of all open sessions in the cluster.
+The **Sessions** page  of the DB Console provides details of all open sessions in the cluster.
```
Comment (maryliag) [inline v21.2/ui-sessions-page.md:12]: this page exists on both DB and CC Console, so I don't think we should be saying only DB Console here (do we even use this term publicly?)

File: v22.1/ui-sessions-page.md (hunk 3 above)
Comment (stbof) [reply v22.1/ui-sessions-page.md:12]: This particular page appears only in the CockroachDB docs: https://deploy-preview-13623--cockroachdb-docs.netlify.app/docs/dev/ui-sessions-page.html

There's a different page for CC Console: https://deploy-preview-13623--cockroachdb-docs.netlify.app/docs/cockroachcloud/sessions-page.html

The pages are constructed from shared components that are included into the parent page.

File: v21.2/ui-sessions-page.md (hunk 4 above)
Comment (stbof) [reply v21.2/ui-sessions-page.md:12]: Yes, we do use the term DB Console publicly: https://deploy-preview-13623--cockroachdb-docs.netlify.app/docs/dev/ui-overview.html

**Review by maryliag — APPROVED** [review]

**Review by rmloveland — APPROVED** [review]
LGTM

**Pull request merged**